src/components/reusable-ui/PrimaryButton.js

```diff
@@ -1,38 +1,45 @@
 import React from "react";
 import styled from "styled-components";
 import { theme } from "../../theme";
+import { IoChevronForward } from "react-icons/io5";
 
 export default function PrimaryButton() {
   return (
-    <PrimaryButtonStyled className="btn">
-      Accédez à mon espace &#62;
+    <PrimaryButtonStyled>
+      <button className="btn">
```

Comment:
```suggestion
      <button>
```
Puisque c'est le seul button de ton composant + ça allège ton JSX. En plus une class "btn" sur un "button" n'apporte pas plus d'info. On va vouloir ajouter une class que si les choses sont confuses (ex: si tu composant contient plusieurs button)